index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header id="header">
+        <div class="logo">
+            <a href="#"><h2>Bilal</h2></a>
+        </div>
+        <div class="hamBurger">
+            <span class="material-symbols-outlined">
+                menu
+                </span>
+        </div>
+    </header>
+    <section id="headline">
+        <h1>
+            I'm M. Bilal Liaquat. Glad to see you!
+        </h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <p class="connect">LET’S CONNECT</p>
+        <ul class="socialIcons">
+            <li><a href="#"><img src="/assets/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="/assets/linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="/assets/medium.svg" alt="medium"></a></li>
+            <li><a href="#"><img src="/assets/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="/assets/victory.svg" alt="victory"></a></li>
+        </ul>
```

Comment:
- [ ] Dear @BilalLiaquat7, For the `LET’S CONNECT` text, we better use an anchor (`a`) tag, Remember this is going to be used to link the users to your contact information later on. 
- [ ] Also for the list of icons, As you can see from the screenshot the image are not visible. This is because you have used `/` instead of `./` to refer for a relative path. Please make sure all the pictures are visible and aligned correctly. Please use an online platform like [githack](https://raw.githack.com/) to check for this issue. 
![image](https://github.com/BilalLiaquat7/Portfolio/assets/69077061/d9ee54a1-f0dc-432e-9324-5be6e1d90b3b)